cmd/puppeth/wizard_genesis.go

```diff
@@ -114,10 +114,6 @@ func (w *wizard) makeGenesis() {
 		}
 		break
 	}
-	// Add a batch of precompile balances to avoid them getting deleted
-	for i := int64(0); i < 256; i++ {
-		genesis.Alloc[common.BigToAddress(big.NewInt(i))] = core.GenesisAccount{Balance: big.NewInt(1)}
-	}
```

Comment:
The reason these accounts are prefunded is to avoid them getting deleted from the state database via the state cleaning EIPs. Please revert this change.

Reply:
thx, I get it.